test/jdk/java/net/Socket/ExceptionText.java

```diff
@@ -104,11 +106,12 @@ static void checkResult(IOException e, boolean withProperty) {
                 throw new RuntimeException("Test failed: exception contains address info");
             }
         } else {
-            if (!msg.contains(HOST) || !msg.contains(PORT)) {
+            if (!msg.contains(HOST) || !msg.contains(PORT) ||
+                !msg.contains(ADDRESS.getHostAddress())) {
                 if (e instanceof ClosedChannelException)
                     return; // has no detail msg
                 System.err.println("msg = " + msg);
-                throw new RuntimeException("Test failed: exception does not contain address info");
+                throw new RuntimeException("Test failed: exception does not contain address info: " + msg);
```

Comment:
It might help debugging any unexpected failures, if the exception message here could include the expected address. Something like:
```
throw new RuntimeException("Test failed: exception message " + msg + " does not contain address " + dest);
```

Reply:
I've added the destination and removed the System.err.println() call above, since it now seems redundant.